medallion/backends/__init__.py

```diff
@@ -0,0 +1,22 @@
+import importlib
+import inspect
+import pkgutil
+import sys
+
+import pkg_resources
+
+from . import base
+
+# Walk this package and import all sub-modules so the can instantiate backends
+_paths = sys.modules[__name__].__path__
+for _, subname, _ in pkgutil.walk_packages(_paths):
+    mod_obj = importlib.import_module(__name__ + "." + subname)
```

Comment:
This needs to be wrapped in a `try, except ImportError` because without installing the `medallion[mongo]` extra `pymongo` won't be installed and importing `mongo_backend.py` will fail.